Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
I think the test would have to fail if there's no toolchain present, and suggest `Pkg.add("WinRPM"); using WinRPM; WinRPM.add("gcc")` the same way `contrib/build_sysimg.jl` does. The fact that we can build the embedding example on the buildbot and distribute the results doesn't seem to serve the purpose of testing that other people can build their own embedding applications? Depending how long it takes to run, I'd think we'd adjust appveyor to do something like we do on Travis, where we run `make install` and make sure the tests execute out of the install tree, rather than the build tree (Travis even moves the build tree to a different name temporarily)